core/src/main/java/org/pentaho/platform/web/http/api/resources/SchedulerResource.java

```diff
@@ -615,6 +615,32 @@ public String doGetCanSchedule() {
     return schedulerService.doGetCanSchedule();
   }
 
+  /**
+   * Checks whether the current user has authority to execute schedules in the platform.
+   *
+   * <p><b>Example Request:</b><br />
+   * GET pentaho/api/scheduler/canExecuteSchedules
+   * </p>
+   *
+   * <p><b>Example Response:</b></p>
+   * <pre function="syntax.xml">
+   *   true
+   * </pre>
+   *
+   * @return true or false. true indicates schedule execution is allowed and false indicates schedule execution is
+   * not allowed for the user.
+   */
+  @GET
+  @Path( "/canExecuteSchedules" )
+  @Produces( APPLICATION_JSON )
+  @StatusCodes( {
+    @ResponseCode( code = 200, condition = "Successful retrieved the scheduling permission." ),
```

Comment:
Might be better to change to `Successfully retrieved` here and for `canSchedule`